catalog/security/catalog-security-filter/src/main/java/ddf/catalog/security/filter/plugin/FilterPlugin.java

```diff
@@ -100,16 +114,15 @@ public QueryResponse process(QueryResponse input)
             }
         }
 
-        logger.info("Filtered {} metacards, returned {}", filteredMetacards,
-                (newResults.size() - filteredMetacards));
+        logger.info("Filtered {} metacards, returned {}", filteredMetacards, newResults.size());
         SecurityLogger.logInfo(
-                "Filtered " + filteredMetacards + " metacards, returned " + (newResults.size()
-                        - filteredMetacards));
+                "Filtered " + filteredMetacards + " metacards, returned " + newResults.size());
 
-        input.getResults().clear();
-        input.getResults().addAll(newResults);
+        input.getResults()
+                .clear();
+        input.getResults()
+                .addAll(newResults);
```

Comment:
Just a comment: To me it's actually a little disconcerting that you can do this .clear() and .addAll() "through" the getResults(), but looking at the impl it should work.